Makefile

```diff
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 
-.PHONY: default clean build shellcheck abcgo golangci-lint style run test cover rest_api_tests rules_content sqlite_db license before_commit openapi-check help install_addlicense install_golangci_lint
+.PHONY: default clean build shellcheck abcgo golangci-lint style run test cover rest_api_tests rules_content sqlite_db license before-commit openapi-check help install-addlicense install-golangci-lint golangci-lint-fix
```

Comment:
I do not think we need 2 targets. I believe we only need one in the makefile, one that runs with fixing. The CI should run without `--fix`, but I do not see the need to do that in the makefile